wmn-data.json

```diff
@@ -2670,7 +2670,7 @@
         "name" : "hiberworld",
         "uri_check" : "https://hiberworld.com/u/{account}",

```

Comment:
This URL is no longer correct and should be: `https://hiberworld.com/user/{account}`